### ISSUE TITLE
[tooltip] Tooltip does not have correct role.

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#177]
The tooltip should have a `role="tooltip"` as described here: https://www.w3.org/TR/wai-aria-1.1/#tooltip
